internal/test/e2e/version_compatibility_test.go

```diff
@@ -211,7 +211,7 @@ func TestVersionCompatibility_PluginFieldUpdates(t *testing.T) {
 	for _, plugin := range expectedPluginsMap {
 		var config structpb.Struct
 		if len(plugin.Config) > 0 {
-			require.NoError(t, json.ProtoJSONUnmarshal([]byte(plugin.Config), &config))
+			require.NoError(t, json.ProtoJSONUnmarshal([]byte(plugin.Config), &config), plugin)
```

Comment:
Adding this to the test fail message output helps isolate where an issue is. For example, I had a trailing comma, so the fail output was:
```
Error:      	Received unexpected error:
        	            	invalid character '}' looking for beginning of object key string
        	Test:       	TestVersionCompatibility_PluginFieldUpdates
        	Messages:   	{Name:zipkin Config:{
        	            				"local_service_name": "LOCAL_SERVICE_NAME",
        	            				"header_type": "ignore",
        	            				"http_span_name": "method_path",
        	            				"connect_timeout": 2001,
        	            				"send_timeout": 2001,
        	            				"read_timeout": 2001,
        	            			} Protocols:[] VersionRange: FieldUpdateChecks:map[< 2.7.0:[{Field:header_type Value:preserve}] >= 3.1.0:[{Field:http_response_header_for_traceid Value:X-B3-TraceId}]] ConfigureForService:true ConfigureForRoute:true ExpectedConfig:}
```

Reply:
Nice one!